src/Snapshooter/MatchOptions.cs

```diff
@@ -86,6 +86,16 @@ public MatchOptions IgnoreFields<T>(string fieldsPath)
             return this;
         }
 
+        public MatchOptions IgnoreAllFieldsByName(string name)
```

Comment:
I think only `IgnoreAllFields` will be enough.

Reply:
Lol... dont review it yet , its not finished yet, I just created a PR to see all the changes I did :)